components/ide-proxy/static/code/profile-templates/profile-templates.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "name": "Python",
+    "url": "https://ide.gitpod.io/code/profile-templates/python.code-profile"
```

Comment:
The current problem is that we need to make sure this domain is correct for the Gitpod installation. I don't think there's an easy way to do that with our current `ide-proxy` setup.

cc @jeanp413